e2e/link_js_package/lib/package.json

```diff
@@ -0,0 +1,3 @@
+{
+    "name": "@e2e/lib"
```

Comment:
nit: "private": True so it's clear this name doesn't exist on npm

Reply:
🌮